internal/ceph/rbd/connection.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
How about moving this under `/pkg/ceph/rbd`? This looks like the last package under `internal`